src/compiler/Stage/Parse/Parser.elm

```diff
@@ -410,23 +410,74 @@ literalInt =
         |> P.inContext InLiteralInt
 
 
-{-| TODO escapes
-TODO Unicode escapes
--}
+-- for literalChar and, in the future, literalString
+stringHelp = 
+    P.oneOf
+    [ P.succeed (identity)
+        |. P.token (P.Token "\\" ExpectingEscapeBackslash)
+        |= P.oneOf
+            [ P.map (\_ -> '\"') (P.token (P.Token "\"" ExpectingEscapeCharacter))
+            , P.map (\_ -> '\'') (P.token (P.Token "'"  ExpectingEscapeCharacter))
+            , P.map (\_ -> '\n') (P.token (P.Token "n"  ExpectingEscapeCharacter))
+            , P.map (\_ -> '\t') (P.token (P.Token "t"  ExpectingEscapeCharacter))
+            , P.map (\_ -> '\r') (P.token (P.Token "r"  ExpectingEscapeCharacter))
+            , P.succeed identity
+                |. P.token (P.Token "u" ExpectingEscapeCharacter)
+                |. P.token (P.Token "{" ExpectingUnicodeEscapeLeftBrace)
+                |= unicode
+                |. P.token (P.Token "}" ExpectingUnicodeEscapeRightBrace)
+            ]
+    ,  P.succeed identity
+        |= P.getChompedString (P.chompIf (always True) ExpectingChar)
+        |> P.andThen (\string ->
+                    string
+                        |> String.uncons
+                        |> Maybe.map (Tuple.first >> P.succeed)
+                        |> Maybe.withDefault (P.problem (CompilerBug "Multiple characters chomped in `literalChar`"))
```

Comment:
This message is not entirely correct, we're not inside `literalChar`. If we want to reuse this in Strings too. it would mislead the user seeing the message.

Reply:
Fixed by https://github.com/elm-in-elm/compiler/pull/20/commits/483bf8a270f7a5bb677bb0e09091de2f58655e78